velox/functions/prestosql/StringFunctions.h

```diff
@@ -52,255 +52,6 @@ struct CodePointFunction {
   }
 };
 
-/// crc32(varbinary) → bigint
-/// Return an int64_t checksum calculated using the crc32 method in zlib.
-template <typename T>
```

Comment:
Can we maybe remove some of the #includes now?

Reply:
Updated.